cobra/test/test_model.py

```diff
@@ -691,6 +691,33 @@ def test_add_reaction_orphans(self, model):
         # 'check not dangling metabolites when running Model.add_reactions
         assert len(orphan_metabolites) == 0
 
+    def test_merge_models(self, model, tiny_toy_model):
+        with model, tiny_toy_model:
```

Comment:
Is the idea that model merges will get reversed after the context exits? That's awesome btw 😄 . Could we add a couple tests to make sure that happens?